Fix HTTParty exception when using both search and query params (#3)
easybroker/easybroker_gem: CHANGELOG.md

## 0.1.4
* Added support for the listing_statuses endpoint.

## 1.0.0
* **Breaking change**: Everytime you make a request `limit`, `page` and `search` no longer should be wrapped inside a `query` key.

> * Since making requests to `www.easybroker.com/api/v1` always redirects to the api subdomain, we updated the root URLs to avoid these redirections.Up

Reviewer: ```suggestion
* Updated endpoints to use the `api` subdomain; `api.easybroker.com/v1` instead of  `www.easybroker.com/api/v1`.
```

## 0.1.5
* Added support for the listing_statuses endpoint for integration partners.

## 1.0.0

> * **Breaking change**: Everytime you make a request `limit`, `page` and `search` no longer should be wrapped inside a `query` key.

Reviewer: ```suggestion
* **Breaking change**: `limit`, `page` and `search` params should be at the root level request instead of in the `query` hash.
```
